addons/web/tooling/_eslintrc.json

```diff
@@ -37,6 +37,12 @@
     "moment": "readonly",
     "py": "readonly",
     "ClipboardJS": "readonly",
-    "globalThis": "readonly"
+    "globalThis": "readonly",
+    "module": "readonly",
+    "chai": "readonly",
+    "describe": "readonly",
+    "it": "readonly",
+    "mocha": "readonly",
+    "DOMPurify": "readonly"
```

Comment:
Confused about this, this is the web tooling's config and I don't see diff in the whitelist in _eslintignore. If you want to use the web tooling in web_editor that's fine but you should probably add the files to the whitelist. If you don't want to use the web tooling and only did this to make runbot green, it's unnecessary, you only need to add them to the test_eslint config.

Reply:
Good catch. We were quite confused about this. Thanks !

Reply:
@Zinston The proper list to update is in `addons/test_lint/tests/eslintrc`.